src/pages/DelegatoryValidator/StakingBar.tsx

```diff
@@ -91,38 +102,69 @@ export default function StakingBar({
     </Button>
   );
 
+  const stakingBarOnMobile = (
+    <Stack direction="column" spacing={4}>
+      <Stack direction="row" spacing={4} justifyContent="space-between">
+        {stakeAmount}
+        {delegatedStakePercentage}
+      </Stack>
+      <Stack direction="row" justifyContent="space-between">
+        {rewardsEarned}
+      </Stack>
+    </Stack>
+  );
+
+  const stakingBarOnWeb = (
+    <Stack direction="row" spacing={4} justifyContent="space-between">
+      <Stack direction="row" spacing={4}>
+        {stakeAmount}
+        {delegatedStakePercentage}
+        <Divider orientation="vertical" flexItem variant="fullWidth" />
+        {rewardsEarned}
+      </Stack>
+      {stakeButton}
+    </Stack>
+  );
+
+  const transactionSucceededDialog = (
+    <TransactionSucceededDialog
+      isDialogOpen={transactionSucceeded}
+      handleDialogClose={handTransactionSucceededDialogClose}
+      stakedAmount={stakedAmount}
+      transactionHash={transactionHash}
+    />
+  );
+
+  const stakeDialog = (
+    <StakeDialog
+      handleDialogClose={handleDialogClose}
+      isDialogOpen={dialogOpen}
+      accountResource={accountResource}
+      validator={validator}
+      setTransactionHash={setTransactionHash}
+      setStakedAmount={setStakedAmount}
+      stakedAmount={stakedAmount}
+      setTransactionSucceeded={setTransactionSucceeded}
+      setTransactionInProcess={setTransactionInProcess}
```

Comment:
they're not used in StakingBar, no need to create the states inside StakingBar and pass by, just move all of them in StakeDialog